src/app/users/users.component.ts

```diff
@@ -166,6 +173,7 @@ export class UsersComponent implements OnInit, AfterViewInit {
           user.roles.push(role);
           const res: any = responses.find((response: any) => response.id === user._id);
           user._rev = res.rev;
+          this.initializeData();

```

Comment:
Do not need to rerun initialize data as we are updating the necessary fields on the user documents